detox/src/realms/DetoxPrimaryContext.test.js

```diff
@@ -66,6 +66,7 @@ describe('DetoxPrimaryContext', () => {
   const detoxWorker = () => latestInstanceOf(DetoxWorker);
   const logFinalizer = () => latestInstanceOf(logger.DetoxLogFinalizer);
   const getSignalHandler = () => lastCallTo(signalExit)[FIRST_ARGUMENT];
+  const facadeInit = () => facade.init({ workerId: null });
```

Comment:
To make sure I don't need to go over each test and think about it, and since the presumption was the most basic init does not install the worker, I added an alias function, `facadeInit` which guarantees that the worker is not spawned.